README.md

```diff
@@ -31,6 +31,15 @@ systems, the system version is too old.  You can pass
 `LIB_PROTO_MUTATOR_DOWNLOAD_PROTOBUF=ON` to cmake to automatically download and
 build a working version of protobuf.
 
+Installation:
+
+```
+sudo ninja install
```

Comment:
does it make sense to do?
```
ninja
sudo ninja install
```

Reply:
Done.

This probably makes sense to avoid people accidentally building targets as root.

Reply:
thanks